ui/app/styles/components/generate-credentials-totp.scss

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+.code-validity {
+  padding-left: $spacing-12;
+  cursor: unset;
```

Comment:
I'm not really sure `unset` is necessary here? We can double check, but ideally it'd be nice to remove this component file and use generic css spacing helpers for the padding